deploy-manage/upgrade/deployment-or-cluster/upgrade-on-eck.md

```diff
@@ -22,6 +22,10 @@ Before you start the upgrade, [plan your upgrade](/deploy-manage/upgrade/plan-up
 
 In this example, we’re modifying the version to {{version.stack}}.
 
+:::{important}
+For production use, configure `vm.max_map_count` to `1048576` in {{es}} version 8.16 and onwards, or `262144` in other verions.
```

Comment:
```suggestion
For production use, configure the `vm.max_map_count` kernel setting to `1048576` in {{es}} version 8.16 and later.
```

Whatever you decide to go with in #4374 should probably be reflected here as well.